awx/ui/test/e2e/commands/navigateTo.js

```diff
@@ -1,8 +1,13 @@
-exports.command = function navigateTo (url) {
+const spinny = 'div.spinny';
+
+exports.command = function navigateTo (url, expectSpinny = true) {
+    this.url('data:,'); // https://github.com/nightwatchjs/nightwatch/issues/1724
```

Comment:
There's a minor bug with the built-in url command with the latest nightwatch version, so this workaround has been added anywhere it was being used.